internal/set/set_test.go

```diff
@@ -30,6 +30,8 @@ func TestAdd(t *testing.T) {
 	// Unrelated elements are unaffected
 	assert.True(t, s.Contains(1))
 	assert.False(t, s.Contains(3))
+	s.AddSlice([]int{3, 4})
```

Comment:
An isolated `TestAddSlice`? Right now the tests are organized mostly one per external function.

Reply:
Done, added a new function.